src/v2/Apps/Settings/Routes/MyCollection/SettingsMyCollectionRoute.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { SettingsMyCollectionRoute_me } from "v2/__generated__/SettingsMyCollectionRoute_me.graphql"
+import { SettingsMyCollectionArtworksQueryRenderer } from "./Components/SettingsMyCollectionArtworks"
+
+interface SettingsMyCollectionRouteProps {
+  me: SettingsMyCollectionRoute_me
+}
+
+const SettingsMyCollectionRoute: React.FC<SettingsMyCollectionRouteProps> = ({
```

Comment:
I mostly copied the code from [SettingsSavesRoute](https://github.com/artsy/force/blob/282d0b6779b2aad7d53f13748dea81695ae4951a/src/v2/Apps/Settings/Routes/Saves/SettingsSavesRoute.tsx#L14).

@dzucconi is there a better/more up-to-date approach I should use instead?

Reply:
Looks good to me.